tasks/components/MobileInterceptor.tsx

```diff
@@ -0,0 +1,33 @@
+import { tw } from '@helpwave/common/twind'
+import type { PropsWithLanguage } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
+import Link from 'next/link'
+import HelpwaveLogo from '../icons/HelpwaveRect'
+
+type MobileInterceptorTranslation = {
+  pleaseDownloadApp: string,
+  playstore: string
+}
+
+const defaultMobileInterceptorTranslation = {
+  en: {
+    pleaseDownloadApp: 'Please download the app',
+    playstore: 'Playstore'
+  },
+  de: {
+    pleaseDownloadApp: 'Bitte laden sie die App herunter',
+    playstore: 'Playstore'
+  }
+}
+
+export const MobileInterceptor = ({ language }:PropsWithLanguage<MobileInterceptorTranslation>) => {
+  const translation = useTranslation(language, defaultMobileInterceptorTranslation)
+  const playstoreLink = 'https://play.google.com/store/apps'
```

Comment:
use next js config

Reply:
Have you resolved this? @DasProffi 
We should get these links from the nextjs config parsing it from the env-vars.

Reply:
Next.js is able to fetch the enviroment parameters set by the underlying system.
https://github.com/helpwave/web/blob/main/tasks/utils/config.ts this is our `config` utility file.
You can define the environment variables inside the `configSchema` prefixed with `NEXT_PUBLIC_`.
Take a look at @MaxSchaefer example in #144